lib/segment/src/vector_storage/dense/mod.rs

```diff
@@ -3,3 +3,5 @@ pub mod dynamic_mmap_flags;
 pub mod memmap_dense_vector_storage;
 pub mod mmap_dense_vectors;
 pub mod simple_dense_vector_storage;
+#[cfg(test)]
+pub mod volatile_dense_vector_storage;
```

Comment:
New storage is marked with `#[cfg(test)]` so that we cannot use it in a release build.